react/official/list/src/pages/ViewMessage.tsx

```diff
@@ -40,7 +40,7 @@ function ViewMessage() {
         {message ? (
           <>
             <IonItem>
-              <IonIcon icon={personCircle} color="primary"></IonIcon>
+              <IonIcon aria-hidden={true} icon={personCircle} color="primary"></IonIcon>
```

Comment:
`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.
```suggestion
              <IonIcon aria-hidden="true" icon={personCircle} color="primary"></IonIcon>
```